scripts/update-delay/recipe_times.py

```diff
@@ -0,0 +1,316 @@
+import re
+import git
+from collections import namedtuple
+from os import environ
+from jug import TaskGenerator, bvalue
+
+RecipeInfo = namedtuple('RecipeInfo', ['pname', 'version', 'url', 'source'])
+
+
+def version_greater(va, vb):
+    '''Compare two versions'''
+    def norm_version(v):
+        return tuple([vb for vb in str(v).split('.')])
+
+    nva = norm_version(va)
+    nvb = norm_version(vb)
+    for ea, eb in zip(nva, nvb):
+        if ea == eb:
+            continue
+        try:
+            return int(ea) > int(eb)
+        except:
+            return ea > eb
+    return False
+
+def extract_meta(meta):
+    '''Tries to parse the contents of a meta.yaml file to build a RecipeInfo
+    object'''
```

Comment:
You can import conda for parsing the meta.yaml. No need to rewrite.

Reply:
I've now spent more time trying to figure out how to update conda-build on the right environment than I had spent reimplementing this.